README.md

```diff
@@ -80,8 +80,8 @@ After completing ALL of the above, remove this `Project bootstrap` section from
 - Update the dependencies management files by performing any number of the following steps:
   - To add a new **frontend** dependency, run `npm install <package name> --save`
     > The above command will update your `package.json`, but won't make the change effective inside the container yet
-  - To add a new **backend** dependency, update `requirements.in` or `dev-requirements.in` with the newest requirements
-- After updating the desired file(s), run `make docker_update_dependencies` to update the containers with the new dependencies
+  - To add a new **backend** dependency, run `docker compose run backend --rm bash` to open an interactive shell and then run `poetry add {dependency}` to add the dependency. If the dependency should be only available for development user append `-G dev` to the command.
```

Comment:
So ALL commands do need to run inside Docker? Note we have two sections below, with and without Docker. If we're forcing always Docker, better to set up devcontainer and update all commands in the README.

Reply:
This is within the `If you are using Docker:` section.